src/commands/exec.ts

```diff
@@ -1,5 +1,6 @@
 import { flags } from '@oclif/command'
 import * as spawn from 'cross-spawn'
+import * as which from 'which'
```

Comment:
Do you know if this is cross platform safe? I couldn't see anything that says yay or nay on the packages npm page

Reply:
https://github.com/npm/node-which/blob/8ef60698398872699b5446a06fc579d281a94b91/which.js#L6 looks like it does.

Totally want to verify this on a windows machine before though 😬

Reply:
It's a dependency of cross-spawn, so it is verified to work on a windows machine 🙂

Reply:
I initially used `spawn.sync('which', [command])` but added the dep for winders

Reply:
Also from the PR description 😜 

> So I utilized a transitive dependency which to perform a cross-platform check